package.json

```diff
@@ -1073,6 +1073,13 @@
           "description": "Zowe-Temp-Folder-Loacation",
           "scope": "window"
         },
+        "Zowe-Builtin-Security": {
+          "default": {
+            "service": ""
+          },
+          "description": "%Zowe-Builtin-Security",
+          "scope": "window"
+        },
```

Comment:
This provide enough customization for the user. :+1:

I'd like to propose a quality-of-life enhancement here (no need to be done now).
Since we support certain services (`@zowe/cli` and `@brightside/core`) I say we could detect whether a profile has secured options by looking at the profile_schema ( properties with `secure:true`) and the property value (assuming it starts with `managed by`)
https://github.com/zowe/imperative/blob/8ca1488259295f4bc40ed5c328c90463f659c86e/packages/profiles/src/constants/ProfilesConstants.ts#L17